src/test/java/com/wikia/webdriver/testcases/portableinfoboxtests/PortableInfoboxTests.java

```diff
@@ -189,15 +189,28 @@ public void verifyHorizontalGroupFontSize() {
 
   @Execute(asUser = User.USER_9)
   public void verifyCopiedTemplateSyntaxInArticlePresence() {
-    TemplatePage template = new TemplatePage();
+//    TemplatePage template = new TemplatePage();
     ArticlePageObject article = new ArticlePageObject();
     PortableInfobox infobox = new PortableInfobox();
-
-    String templateSyntax =
+    String templateSyntax = "<infobox layout>\n"
+                                          + "<image source=\"image\"><default>\n"
+                                          + "[[File:Burton-jane-ginger-domestic-kitten-felis-catus-rolling-on-back-playing.jpg|thumb|sweet kitty]]\n"
+                                          + "</default><alt>I AM ALT :)</alt>\n"
+                                          + "<caption source=\"z1\"><format>[[{{{z1}}}$$]]</format><default>Default image caption</default></caption>\n"
+                                          + "</image>\n"
+                                          + "                                                                                <title source=\"name\"><default>thisAppearsIfNoNameForInfobox</default></title>\n"
+                                          + "<title source=\"name2\"><default>This is second title</default></title>\n"
+                                          + "<data source=\"url\"><label>\"URL\"</label></data>\n"
+                                          + "<data source=\"nolabel\"></data>\n"
+                                          + "<data source=\"url2\"><label></label><default>\"URL2222222222222\"</default></data>\n"
+                                          + "<data source=\"url3\"><label>\"URL33433333333\"</label></data>\n"
+                                          + "<data source=\"url222222222\"><label>URL</label><default>DEFAULT VALUE</default></data>\n"
+                                          + "</infobox>\n";
+   /* String templateSyntax =
         template
             .openArticleByName(wikiURL, PageContent.PI_TEMPLATE_WEBSITE_SIMPLE)
             .editArticleInSrcUsingDropdown()
-            .copyContent();
+            .copyContent();*/
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3ACommentedOutCodeLine)